### PR TITLE
Combining opensim-visualize and opensim-cmd

### PR DESCRIPTION
**Pull request by chrisdembia**
This is a draft of combining the `opensim-visualize` and `opensim-cmd` executables, to spark discussion about the combined executable.

In testing this executable, I found that `VisualizerUtilities::showMarkerData()` does not work for me.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2777)

### CODE REVIEW

**Comment by aymanhab** [comment]
Considering where we are @chrisdembia I'd love to get this moving. I can merge this PR into the branch to simlify the review/merge process unless you're actively working on it. Thank you

**Comment by chrisdembia** [comment]
I'm not actively working on this branch; your thoughts on my responses to my review could help me move forward and wrap this up.

**Comment by chrisdembia** [comment]
@aymanhab the diff is messed up but I pushed the changes you mentioned. Maybe later today I can fix the diff.

**Comment by aymanhab** [comment]
No worries @chrisdembia I can take over when you give me the green light.

**Comment by chrisdembia** [comment]
You have the green light!

File: Applications/opensim-cmd/opensim-cmd_viz.h [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+ *                      OpenSim:  opensim-cmd_viz.h                           *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2017 Stanford University and the Authors                *
+ * Author(s): Frank C. Anderson, Ayman Habib, Chris Dembia                    *
```
Comment (aymanhab) [inline Applications/opensim-cmd/opensim-cmd_viz.h:13]: Year and Authors need update 😃

**Comment by aymanhab** [comment]
Thanks @chrisdembia I fixed upstream, tested locally, added option to rotate quaternion data per Johanna. Feel free to re-review or merge, as she needs it rather soon.

**Comment by aymanhab** [comment]
@chrisdembia thanks for the review, adding the detailed option triggers the regex error, I didn't bother since this is experimental so I didn't want to update the documentation until finalized (but I will update per your feedback) and will see if the issue reproduces on ci.

**Comment by aymanhab** [comment]
Thanks @chrisdembia based on addressing your comments on documentation, this is ready to merge unless I hear otherwise. Thanks again for making this PR much better 👍

**Comment by chrisdembia** [comment]
LGTM

**Pull request merged**